BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollector.java

```diff
@@ -36,33 +36,35 @@ public DataCollector(@NonNull BraintreeClient braintreeClient) {
      * @param context  Android Context
      * @param callback {@link DataCollectorCallback}
      */
-    public void collectDeviceData(@NonNull Context context, @NonNull DataCollectorCallback callback) {
-        collectDeviceData(context, null, callback);
+    public void collectDeviceData(
```

Comment:
Can be future PR but we should also deprecate this if we want merchants to pass in the request every time

Reply:
👍 let me add the deprecation annotation